app/server/models/User.js

```diff
@@ -40,7 +40,7 @@ var profile = {
 
   essay: {
     type: String,
-    min: 0,
+    min: 100,
```

Comment:
lets make this zero just so we don't break out mongo tables by accident

Reply:
I reverted this back to 0. the change wasnt committed